crates/bevy_ecs/src/removal_detection.rs

```diff
@@ -83,6 +83,11 @@ impl RemovedComponentEvents {
         }
     }
 
+    /// Returns iterator over components and their entity events.
```

Comment:
```suggestion
    /// Returns iterator an over components and their entity events.
```

Reply:
Maybe you wanted to write `an iterator`?

Reply:
Yep, top 10 epic fails